client/solo/commands/_manage.py

```diff
@@ -737,10 +737,14 @@ def _get_license_text(license_name, repository_id):
         # that the user accepts their licenses
         licenses = entropy_client.get_licenses_to_accept(
             package_matches)
+
         # ACCEPT_LICENSE env var support
-        accept_license = os.getenv("ACCEPT_LICENSE", "").split()
```

Comment:
just do:
if "*" in accept_license:
    licenses.clear()
else:
    for mylic in accept_license:
        licenses.pop(mylic, None)